src/api.rs

```diff
@@ -13,4 +13,6 @@ pub struct CaptureRequest {
 }
 
 #[derive(Debug, Deserialize, Serialize)]
-pub struct CaptureResponse {}
+pub struct CaptureResponse {
+    pub status: u8,
```

Comment:
Let's make this mean something, perhaps an enum with only one possible value or... something